packages/core/src/utils/PackageJson.ts

```diff
@@ -1,21 +0,0 @@
-import * as t from "io-ts";
```

Comment:
I just remember now that Arnaud told me that they use this type in the playground to parse the package.json, I think we should check with the playground team before removing this type completely. 

I still agree that we should remove it and use the one define in the node modules :)

Reply:
ooooh

Reply:
I moved the type to Models so maybe that'll be enough, I'll ask

Reply:
I found these 2 files that use the core:
- https://github.com/prismicio/playground-libraries/blob/6d1d260fe3d91f83afaebf365f7224d323658755/src/models/Library.ts
- https://github.com/prismicio/playground-libraries/blob/df14d90433c5f253d9db6a8a249642e3f8ce166a/src/MainHandler.ts

but no instance of the PackageJson helper

Reply:
here's the parsing of the package: https://github.com/prismicio/playground-libraries/blob/df14d90433c5f253d9db6a8a249642e3f8ce166a/src/MainHandler.ts#L58